tests/dummy/app/adapters/application.js

```diff
@@ -1,4 +1,4 @@
 import DS from 'ember-data';
 
-export default DS.JSONAPIAdapter.extend();
-//export default DS.RESTAdapter.extend();
+// export default DS.JSONAPIAdapter.extend();
+export default DS.RESTAdapter.extend();
```

Comment:
Obviously you wouldn't want to do this for all the tests but just wanted to turn it on here to show the failures